brave-cxf/src/main/java/com/github/kristofa/brave/cxf/MessageWrapper.java

```diff
@@ -0,0 +1,64 @@
+package com.github.kristofa.brave.cxf;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.apache.cxf.message.Message;
+
+/**
+ * @author Michał Podsiedzik
+ */
+public class MessageWrapper {
+    protected final Message message;
+
+    public MessageWrapper(final Message message) {
+        this.message = message;
+    }
+
+    protected URI getUri() {
+        try {
+            return new URI((String) message.getExchange().get(Message.ENDPOINT_ADDRESS));
+        } catch (URISyntaxException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    protected void addHeader(final String header, final String value) {
```

Comment:
all of these methods are used inside the same package, so make them package protected